src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj.BAK

```diff
@@ -12,7 +12,7 @@
     <DefineConstants>$(DefineConstants);NETCOREAPP1_0;TRACE</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PreserveCompilationContext>true</PreserveCompilationContext>
-    <DebugType>portable</DebugType>
+    <DebugType>full</DebugType>
```

Comment:
why are we checking in the .BAK files that get generated during code coverage runs.  Please remove all extraneous content from the PR.

Reply:
@NiranjanVirtuosity: can you address these really old comments? Please, try to be diligent in what you submit. There seems to be quite a bit of junk in your commits...